doc/usage.md

```diff
@@ -320,3 +320,61 @@ oc run --rm -i --restart=Never --image=registry.fedoraproject.org/fedora-minimal
 rer $(cat /var/run/secrets/kubernetes.io/serviceaccount/token)" https://metrics.openshift-compliance.svc:8585/metrics-co' | grep compliance
 ```
 
+## To use PriorityClass for scans
+
+When heavily using Pod Priority and Preemption for automated scaling and
```

Comment:
We could link Pod Priority and Preemption to the documentation you linked below.

https://kubernetes.io/docs/concepts/scheduling-eviction/pod-priority-preemption/